### ISSUE TITLE
DDB tests don't check all tables

### ISSUE THREAD

**Issue by RAR** [sensu-plugins/sensu-plugins-aws#191]
We have 415 or so ddb tables but when I run the ddb check scripts (either throttle or capacity) they stop at 100; I'm assuming pagination needs to be added.

**Comment by majormoses** [comment]
yes, we could look at doing something similar to what I did over here: https://github.com/sensu-plugins/sensu-plugins-aws/pull/207 would you be willing to put together a pr?
